code/datums/helper_datums/synthsprites.dm

```diff
@@ -0,0 +1,69 @@
+
+/* 
+
+Just some quick documentation about how this works
+
+Synthckey is the players ckey which is represented under the synths name in the json, this is required to have
+synthicon is just the sprites name in the dmi
+ainame currently is not normaly used unless there is a rare case where a synths name is identical to the ai name but needs to be seperated
```

Comment:
Do not merge dead code. There's no code to use `ainame` ergo it should be removed.